sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -222,6 +223,6 @@ export function toProcessingSpanOptions(
   return {
     spanLinks,
     spanKind: "consumer",
-    ...toSpanOptions(eventHubProperties),
+    ...toSpanOptions(eventHubProperties, operation),
```

Comment:
Do we expect the operation to be something different from "process" anywhere? If not, may be we can hard code it here rather than making it a param.

Reply:
Hard coded to "process" only